common/test/acceptance/tests/lms/test_lms_cohorted_courseware_search.py

```diff
@@ -57,15 +58,15 @@ def setUp(self, is_staff=True):
         self.content_group_b = "Content Group B"
 
         # Create a student who will be in "Cohort A"
-        self.cohort_a_student_username = "cohort_a_student"
-        self.cohort_a_student_email = "cohort_a_student@example.com"
+        self.cohort_a_student_username = "cohort_a_student" + "_" + str(uuid.uuid4().hex)[:5]
```

Comment:
I seem to recall a test that was occasionally failing before that had too small a random pool to choose from, but cannot remember if it was 3 characters or 5. I'd suggest we get rid of the word "student" and instead add 7 characters to the uuid part. I.e. .hex)[:12]